src/main/java/de/tum/in/www1/artemis/service/compass/CompassCalculationEngine.java

```diff
@@ -363,6 +371,24 @@ public JsonObject getStatistics() {
         return jsonObject;
     }
 
+    @Override
+    public double getConfidenceForElement(String elementId, long submissionId) {
```

Comment:
JavaDoc missing

Reply:
The JavaDoc for this method can be found in the `CalculationEngine` interface. I extended it a bit.